packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -723,39 +729,18 @@ export const RemoteServerFactory = {
                     // purpose so we don't want to overwrite the header for those calls.
                     if (incomingRequest.path?.match(options.applySLASPrivateClientToEndpoints)) {
                         proxyRequest.setHeader('Authorization', `Basic ${encodedSlasCredentials}`)
+                    } else if (!incomingRequest.path?.match(options.slasApiPath)) {
```

Comment:
can we add unit test to cover this?

Reply:
We don't have a good way to write a unit test for this as this logic is inside the proxy. We'd need to have the proxy (and therefore the app as a whole) up and running. 

The simplest way to test this is via manual test. Longer term, we could write a test script that runs various curl commands against a deployed CI environment and check for its responses.